WordPress/gutenberg
# [RNMobile] Adjust FloatingToolbar to first shippable temporary version

## Description
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1413
Ref to gb-mobile repo: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1419
## How has this been tested?

Tested on both platforms:

1. Add a `Group`
2. Add any block inside a `Group`
3. Notice the `FloatingToolbar`

## Screenshots 
Will be added soon.  --> added below.

## Types of changes
Render FloatingToolbar over the item but without overlapping.

## Checklist:
